pkg/process/util/config_fallback.go

```diff
@@ -0,0 +1,15 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build !linux
+
+package util
+
+import "github.com/DataDog/datadog-agent/pkg/config/model"
+
+// GetRunInCoreAgentConfig returns false in place of the config value for process_config.run_in_core_agent.enabled.
+func GetRunInCoreAgentConfig(_ model.Reader) bool {
+	return false
+}
```

Comment:
Why not enforce the setting `process_config.run_in_core_agent.enabled` to false when running on windows ? This could be done either in the init of on of the component in `comp/process`.

Reply:
I wasn't aware that was a possibility 😅 Whats the best method to do this? `config.SetWithoutSource`?

Reply:
I think I got it, PTAL.

Reply:
FYI, couldn't get this override at a comp-level as the checks are initialized with the [config](https://github.com/DataDog/datadog-agent/blob/818ca548990dcd039e4c3003c23b09f2173b0a77/pkg/process/checks/process.go#L53) before the process agent component itself.